wasm/src/account/signature.rs

```diff
@@ -76,6 +77,20 @@ impl Signature {
     pub fn to_string(&self) -> String {
         self.0.to_string()
     }
+
+    /// Get the left endian byte array representation of the signature plaintext.
+    #[wasm_bindgen(js_name = "toPlaintextBytesLe")]
+    pub fn to_plaintext_bytes_le(&self) -> Result<Uint8Array, String> {
+        let plaintext = Plaintext::from(LiteralNative::Signature(Box::new(self.0)));
+        plaintext.to_bytes_le()
+    }
+
+    /// Get the left endian boolean array representation of the signature plaintext bits.
+    #[wasm_bindgen(js_name = "toPlaintextBitsLe")]
+    pub fn to_plaintext_bits_le(&self) -> Array {
+        let plaintext = Plaintext::from(LiteralNative::Signature(Box::new(self.0)));
+        plaintext.to_bits_le()
+    }
```

Comment:
Can you add `fromBytesLe`, `toBytesLe`, `fromBitsLe` `toBitsLe`, `toPlaintext`, `toFields`, `fromFields` on this  type as well?